site/content/recipes/perform_code_review.md

```diff
@@ -37,14 +37,35 @@
 
 ## Reviewer reviews code
 
-1. Reviewer reminds themselves of objectives and areas of focus from first meeting (link to Meet and Agree on Objectives section)
+What should you do if you're the reviewer, providing feedback on someone else's
+code?
+We suggest a workflow with two key elements.
+First of all, keep the authors' objectives in mind.
+To achieve this, begin your review by refreshing your memory about what you and
+the author discussed when you met initially.
+The second key element is to have a checklist.
```

Comment:
Reviewing code can be a difficult exercise. Where do you even start? What do you look at? The good news is: there are a number of points that are _almost always_ worth looking at. Particularly, readability is often a good angle from which starting a code review. You can start simple, for instance looking at variables and functions' name: are they explicit?  Do they carry intent? As you get more and more familiar with the code under review, move on to larger structure: can some of the functions be split into several functions? Do you see any complex conditional statements that you find hard to read? Eventually, perhaps after several iterations of the code review process, you may be able to criticise the code design, for instance its modules and the relationships between them. But don't start from there!